README.md

```diff
@@ -10,6 +10,7 @@ If you've used Hardhat 👷‍♀️👷‍♂️ and want to develop for Starkn
 -   [Testing](#test)
     -   [Important notes](#important-notes)
     -   [Examples](#test-examples)
+    -   [Devnet examples](#devnet)
```

Comment:
The link is dead, they can be tested even in vs-code (it should be `#devnet-examples`)

Reply:
[ea3c5bf](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/89/commits/ea3c5bf494389944fb7c1db77fed48e343fe8fa0)